tests/integration/medusa_minio_integration_test.go

```diff
@@ -0,0 +1,61 @@
+package integration
+
+import (
+	. "github.com/k8ssandra/k8ssandra/tests/integration/steps"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var (
+	medusaMinioTestTable    = "medusa_test"
+	medusaMinioTestKeyspace = "medusa"
+)
+
+var _ = Describe("Medusa on MinIO", func() {
+	Context("Deploy K8ssandra with Medusa", func() {
+		It("Create a kind cluster and a namespace", func() {
+			AKindClusterIsRunningAndReachableStep("one worker")
+			IInstallTraefikStep()
+			ICreateTheNamespaceStep()
+			ICanSeeTheNamespaceInTheListOfNamespacesStep()
+		})
+		It("Create the Medusa secret ", func() {
+			IDeployMinIOUsingHelmAndCreateTheBucketStep("k8ssandra-medusa")
```

Comment:
These function names are too-Springish. Go prefers short function names.

Reply:
If we go with a less descriptive approach to our testing, for example removal of Ginkgo, the function naming will become critical to understanding any of what is going on.  I suspect we will have to have long function names and/or numerous comments.

Reply:
While this is subjective I tend to agree that the function names do not seem like very idiomatic Go. 

We should aim to have descriptive function names regardless of whether we use Ginkgo, testing, or something else. 

Readable, descriptive test code is definitely important, but I feel we are getting hung up some things that feel a bit academic right now. I would like to see more discussion and focus around things like:

* How do setup/tear down things?
* How should we verify various state?
* What clients should we use to talk to the api server?
* How do we run things in parallel for CI?

Reply:
These functions are inherited from the GoDog experiments which auto-generated function names, I didn't named them myself ;)
We can definitely make them shorter, but it would indeed be good to keep them descriptive enough (Go or not Go, I think that should apply to all languages).

Reply:
Long doesn't always indicate descriptive as it makes reading them a lot slower. Also it should not have information that's obvious.